heka/sandbox/filters/telemetry_decoder_view.lua

```diff
@@ -14,7 +14,9 @@ require "string"
 local SEC_PER_ROW   = 60
 local ROWS          = 2880
 
-bf                  = bloom_filter.new(3*1e6, 0.01) -- todo up the size when not under the sandbox manager
+local items         = read_config("bloom_items") or 3*1e6
```

Comment:
I just blew away the filter state manually when I redeployed, but yes, this is the correct approach.